firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/FirebaseAppDistribution.java

```diff
@@ -170,6 +178,10 @@ public UpdateTask updateIfNewReleaseAvailable() {
     }
   }
 
+  private boolean notOnMainThread() {
```

Comment:
Method doesn't do much so I can inline it. My initial thought would be that giving it a name could clarify what we are checking.

Reply:
I'd recommend renaming this method so the boolean represents the positive check (isOnMainThread), rather than the negative check. This helps prevent double negatives, which can be tricky to read (More about this stylisitic guide here: go/tott/530)

Reply:
I was thinking of my last PR https://github.com/firebase/firebase-android-sdk/pull/3259#discussion_r773464338 In the spirit of not negating the return value thoughts on isOnBackgroundThread?

Reply:
That sounds good too